events/events.h

```diff
@@ -5,10 +5,11 @@
 
 /**
  * events_immediate_register(func, cookie, prio):
- * Register ${func}(${cookie}) to be run the next time events_run is invoked,
- * after immediate events with smaller ${prio} values and before events with
- * larger ${prio} values.  The value ${prio} must be in the range [0, 31].
- * Return a cookie which can be passed to events_immediate_cancel.
+ * Register ${func}(${cookie}) to be run the next time events_run or
+ * events_spin is invoked, after immediate events with smaller ${prio} values
```

Comment:
I'm no longer confident that we need to explicitly mention events_spin here.  A few weeks ago, I was 90% certain this was a good change; now I'm hovering around 60%.

Reply:
I don't think it really helps.  If you want to clarify things, I'd add a note in `events_spin` saying that it uses `events_run`.